controller/controller_manager.go

```diff
@@ -46,6 +46,9 @@ func StartControllers(logger logrus.FieldLogger, stopCh chan struct{}, controlle
 		return nil, nil, errors.Wrap(err, "unable to get client config")
 	}
 
+	config.Burst = 100
+	config.QPS = 50
```

Comment:
For reference the Default values were:
```
	DefaultQPS   float32 = 5.0
	DefaultBurst int     = 10
```
https://github.com/longhorn/longhorn-manager/blob/24d07f134011e8ef2ad01d2d69427a59c43ed87c/vendor/k8s.io/client-go/rest/config.go#L44-L45